gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/TestHelper.java

```diff
@@ -50,6 +52,7 @@
 public final class TestHelper {
 
     private static final String SEP = File.separator;
+    private static final char URL_SEP = '/';
```

Comment:
ah! the class lookup is by URL so you need standard URL pathing to handle that (and not the windows file separator). cool